tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1537,3 +1537,64 @@ function assertIncludesMessage(messageArr, message) {
     return false
   }
 }
+
+Then('the decline action button should be visible on the shared-with-me page', function () {
+  return batchActionButtonForSharedWithMePageIsDisplayed()
+})
+
+const batchActionButtonForSharedWithMePageIsDisplayed = function () {
+  const batchActionButtonForSharedPageXpath =
+    client.page.sharedWithMePage().elements.batchDeclineSharesButton.selector
+  return client.expect.element(batchActionButtonForSharedPageXpath).to.be.present
+}
+
+Then('the delete action button should be visible on the shared-with-others page', function () {
+  return batchActionButtonForSharedWithOtherPageIsDisplayed()
+})
+
+const batchActionButtonForSharedWithOtherPageIsDisplayed = function () {
+  const batchActionButtonForSharedWithOtherPageXpath =
+    client.page.sharedWithOthersPage().elements.batchDeleteSharesButton.selector
+  return client.expect.element(batchActionButtonForSharedWithOtherPageXpath).not.to.be.not.present
+}
+
+When('the user selects the file/folder {string} using the webUI', function (fileName) {
+  return client.page.FilesPageElement.filesList().toggleFileOrFolderCheckbox('enable', fileName)
+})
+
+Then(
+  'the following batch action buttons should be visible on the shared with me page',
+  async function (dataTable) {
+    const batchButtons = dataTable.hashes()
+    for (const batchButton of batchButtons) {
+      const isBatchButtonExist = await client.page
+        .sharedWithMePage()
+        .isBatchButtonVisible(batchButton.buttons)
```

Comment:
nice implementation 👍🏼 
```suggestion
      const isBatchButtonVisible = await client.page
        .sharedWithMePage()
        .isBatchButtonVisible(batchButton.buttons)
```